admin_guide/high_availability.adoc

```diff
@@ -496,45 +457,30 @@ ifdef::openshift-origin[]
 ====
 ----
 $ oadm ipfailover ipf-ha-geo-cache \
-    --replicas=5 --watch-port=9736 \
-    --selector="ha-svc-nodes=geo-us-west" \
-    --virtual-ips=10.245.3.101-105 \
-    --vrrp-id-offset=10 \
+    --replicas=4 --selector="ha-cache=geo" \
+    --virtual-ips=10.245.2.101-104 --watch-port=9736 \
     --service-account=ipfailover --create
 ----
 ====
 endif::[]
 ////
 +
-In the commands above, there are *ipfailover*, *router*, and *geo-cache* pods on
-each node. The set of VIPs for each ipfailover configuration must not overlap
-and they must not be used elsewhere in the external or cloud environments. The
-five VIP addresses in each example, `10.245.{2,3}.101-105` are served by the two
-ipfailover deployment configurations. IP failover dynamically selects which
-address is served on which node.
-+
-The administrator sets up external DNS to point to the VIP addresses knowing
-that all the *router* VIPs point to the same *router*, and all the *geo-cache*
-VIPs point to the same *geo-cache* service. As long as one node remains running,
-all the VIP addresses are served.
-
-. Deploy the ipfailover router to monitor postgresql listening on node
-port 32439 and the external IP address, as defined in the *postgresql-ingress*
-service:
+As an alternative, the following example creates an IP failover configuration on
+a selection of nodes labeled "my-ha-service=har-reporter" (on 4 nodes with 7
+VIPs monitoring a service listening on port 4242:
 +
 ====
 ----
-$ oadm ipfailover ipf-ha-postgresql \
-    --replicas=1 <1> --selector="app-type=postgresql" <2> \
-    --virtual-ips=10.9.54.100 <3> --watch-port=32439 <4>  \
-    --credentials=/etc/origin/master/openshift-router.kubeconfig \
-    --service-account=ipfailover --create
+$ oadm ipfailover harreporter --selector="my-ha-service=har-reporter" --virtual-ips="10.245.2.42,10.245.2.100-104,10.245.2.142,10.245.2.242" --watch-port=4242 --replicas=7 --service-account=ipfailover --create
 ----
-<1> Specifies the number of instances to deploy.
-<2> Restricts where the ipfailover is deployed.
-<3> Virtual IP address to monitor.
-<4> Port on which ipfailover will monitor on each node.
 ====
+////
+
+Using the above example, you can now use the VIPs 10.245.2.101 through
+10.245.2.104 to send traffic to the geo-cache service. If a particular geo-cache
+instance is "unreachable", perhaps due to a node failure, Keepalived ensures
```

Comment:
If the instance becomes not reachable at a given VIP (the watch port fails), keepalived will assign the VIP to another replica in the set. The app will be reachable as long as 1 keepalived is running. (This means that all of the VIPs can end up on the same node)